client/pokemon/pokemon-full.view.html

```diff
@@ -171,6 +171,7 @@ <h1 class="summary-toolbar">{{pokemon.parsedNickname}}</h1>
                   <span flex>Stat</span>
                   <span flex>Base</span>
                   <span flex>IVs</span>
+                  <span flex ng-if="pokemon.gen === 7">H/T</span>
```

Comment:
I was thinking tooltip if it doesn't look too terrible. That way it's more obvious and I believe that'd keep it usable on mobile, would it not?